profile/README.md

```diff
@@ -1,48 +1,40 @@
 
-With the Camunda Community Hub we want to empower our open source community to contribute, and encourage new contributors to get started contributing to the Camunda open source ecosystem.
+With the Camunda Community Hub, we want to empower our open source community to contribute, and encourage new contributors to get started contributing to the Camunda open source ecosystem.
 
-## Existing community extensions
-
-You can find a list of existing community extensions by [browsing the repostories](https://github.com/orgs/camunda-community-hub/repositories) in this GitHub organization.
+**Table of Contents**
 
-## FAQ and further information
+* [Existing community extensions](#existing-community-extensions)
+* [How to contribute](#how-to-contribute)
+* [Troubleshooting](#troubleshooting)
 
-We’ve put together some commonly asked questions and answers on https://github.com/camunda-community-hub/community. If you have further questions, you can also ask via our [forum](https://forum.camunda.io/) or reach out to the [Camunda DevRel team](https://github.com/orgs/camunda-community-hub/teams/devrel) via email: community@camunda.com.
+## Existing community extensions
 
-## How to get my community extension to the Camunda Community Hub?
+You can find a list of existing community extensions by [browsing the repostories](https://github.com/orgs/camunda-community-hub/repositories) in this GitHub organization.
 
-- [Open an issue](https://github.com/Camunda-Community-Hub/community/issues/new/) in the [community repository](https://github.com/Camunda-Community-Hub/community/) and ask to be invited to join the Camunda Community Hub (if you are not already a collaborator).
+You can also browse by Awesome Lists created by the community.
+- [Awesome Camunda External Task Clients](https://github.com/camunda-community-hub/awesome-camunda-external-clients)
+- [Awesome Camunda Platform 8 Projects](https://github.com/camunda-community-hub/awesome-camunda-platform-8)
+- [Awesome Camunda and Robot Framework Projects](https://github.com/camunda-community-hub/awesome-Camunda-and-Robotframework-projects)
 
-- Sign our [CLA](https://cla-assistant.io/camunda-community-hub/community) and agree to our [Code of Conduct](https://camunda.com/events/code-conduct/). 
+## How to contribute
+- [Open an issue](https://github.com/Camunda-Community-Hub/community/issues/new/) in the [community repository](https://github.com/Camunda-Community-Hub/community/) and ask to be invited to join the Camunda Community Hub (if you are not already a member).
+- Read our [Contributor's Guide](https://github.com/camunda-community-hub/community/blob/a6526a0f07893aa84434a2cadd9ffdc332a1f89d/CONTRIBUTING.MD#L25).
```

Comment:
This link looks like it includes a commit or blob URL. I would use this one instead - `https://github.com/camunda-community-hub/community/blob/main/CONTRIBUTING.MD`.